packages/wp-dashboard/src/api/utils/reshapeStoryObject.js

```diff
@@ -70,10 +71,11 @@ export default function reshapeStoryObject(originalStoryData) {
       avatar: lockUser[0]?.avatar_urls?.['96'] || null,
     },
     bottomTargetAction: editStoryLink,
-    featuredMediaUrl,
+    featuredMediaUrl: poster?.url ? poster?.url : featuredMediaUrl,
```

Comment:
Nit: the optional chaining should not be needed because of the default value of `{}`.

Then could simply do:

```suggestion
    featuredMediaUrl: poster.url || featuredMediaUrl,
```